docs/develop/glossary.md

```diff
@@ -1,57 +0,0 @@
----
-sidebar_position: 3
-
----
-
-# Glossary
```

Comment:
this is probably because its not on the sdk-repo, its a new piece

Reply:
dont merge yet im going through each piece

Reply:
no its not in main either, seems it was removed outright. looks useful

Reply:
Yes, the scripts simply copies the learn, develop and build category from the repo.
In my opinion, the glossary is useful for each version and common and does not need to be versioned.
This file could be moved out of develop, build or learn category and be common to all versions.

Reply:
Yeah i wrote the glossary and only put it in the docs repo so this is something that could be excluded from the whole duplication of docs

Reply:
Yeah, the file should just be moved the `docs` folder directly, https://github.com/cosmos/cosmos-sdk-docs/tree/main/docs, so need of subfolder.
This may be relevant: https://docusaurus.io/docs/docs-multi-instance, so the glossary (and possibly tutorials?) aren't versioned but common to all versions.